### PR TITLE
Want to fail if there is an error in importing or exporting

### LINKED ISSUE #10: Fail loudly if there are malformed keys
- We don't want to write bad data. What should happen is that we fail loudly if there are malformed keys and report the K/V path of where the error is.

### PR DESCRIPTION
**Pull request by abhiyerra**
Closes #10

### CODE REVIEW

File: consul.go [hunk 1]
```diff
@@ -118,7 +127,8 @@ func init() {
 
 	client, err = api.NewClient(api.DefaultConfig())
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(-1)
```
Comment (parkr) [inline consul.go]: `fmt.Fatalln` has the same effect as `Println` and `Exit`

File: main.go [hunk 2]
```diff
@@ -37,17 +37,26 @@ func main() {
...
+			os.Exit(-1)
+		}
 	case "import":
 		jsonImport := &JsonImport{}
 		jsonImport.ParseFlags(os.Args[2:])
-		jsonImport.Run()
+		err := jsonImport.Run()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(-1)
```
Comment (parkr) [inline main.go]: same here with `fmt.Fatalln`

File: main.go [hunk 3]
```diff
@@ -37,17 +37,26 @@ func main() {
 	case "export":
 		jsonExport := &JsonExport{Watch: false}
 		jsonExport.ParseFlags(os.Args[2:])
-		jsonExport.Run()
+		err := jsonExport.Run()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(-1)
```
Comment (parkr) [inline main.go]: same here with `fmt.Fatalln`

File: main.go [hunk 4]
```diff
@@ -37,17 +37,26 @@ func main() {
...
+		}
 	case "set":
 		jsonSet := &JsonSet{}
 		jsonSet.ParseFlags(os.Args[2:])
 		err := jsonSet.Run()
 		if err != nil {
 			fmt.Println(err)
+			os.Exit(-1)
```
Comment (parkr) [inline main.go]: same here with `fmt.Fatalln`

**Comment by parkr** [comment]
nits :)

File: consul.go (hunk 1 above)
Comment (abhiyerra) [inline consul.go]: Thanks. I was wondering about that.

**Pull request merged**